Docker/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+cd ${WORKDIR}
+
+
+if [ -f "${DATADIR}/config.ini" ]
+  then
+    echo
+  else
+    cp /config.ini ${DATADIR}
+fi
+
+
+if [ "${NODE_TYPE}" = "test" ]; then
+	if [ -f "${DATADIR}/genesis-test.json" ]
+	  then
+	    echo "Starting muse daemon in TEST"
+	  	exec mused -s ${TEST_SEED} \
+		--rpc-endpoint=0.0.0.0:8090 \
+		--genesis-json ${DATADIR}/genesis-test.json \
+		-d ${DATADIR}/
+	  else
+	  	echo "Starting muse daemon in TEST.  Replaying blockchain"
+	    cp /genesis-test.json ${DATADIR}
+	    exec mused -s ${TEST_SEED} \
+		--replay-blockchain --rpc-endpoint=0.0.0.0:8090 \
+		--genesis-json ${DATADIR}/genesis-test.json \
+		-d ${DATADIR}/
+	fi
+
+fi
+
+if [ "${NODE_TYPE}" = "prod" ]; then
```

Comment:
For production, genesis json is embedded, see egenesis_full.cpp
Using genesis.json is not required.

Reply:
right!

Reply:
I'll remove.